code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1101,14 +1128,22 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			if(I.flags_2 & SLOWS_WHILE_IN_HAND_2)
 				. += I.slowdown
 		var/health_deficiency = (100 - H.health + H.staminaloss)
-		var/hungry = (500 - H.nutrition) / 5 // So overeat would be 100 and default level would be 80
 		if(health_deficiency >= 40)
 			if(flight)
 				. += (health_deficiency / 75)
 			else
 				. += (health_deficiency / 25)
-		if((hungry >= 70) && !flight)		//Being hungry won't stop you from using flightpack controls/flapping your wings although it probably will in the wing case but who cares.
-			. += hungry / 50
+
+		GET_COMPONENT_FROM(mood, /datum/component/mood, H)
+		if(mood && !flight) //How can depression slow you down if you can just fly away from your problems?
+			switch(mood.mood)
+				if(-INFINITY to MOOD_LEVEL_SAD4)
+					. += 1.5
+				if(MOOD_LEVEL_SAD4 to MOOD_LEVEL_SAD3)
+					. += 1
+				if(MOOD_LEVEL_SAD3 to MOOD_LEVEL_SAD2)
+					. += 0.5
```

Comment:
God no.

Reply:
wahhh wahhghh WAAAAAAAAH

it replaces food shut up

Reply:
Hunger slowdown is removed and this is arguably more difficult to reach than being hungry.

Reply:
@kevinz000 stop whining

Reply:
@kevinz000 stop whining

Reply:
@kevinz000 stop whining

Reply:
@kevinz000 keep whining

Reply:
@kevinz000 keep whining

Reply:
@kevinz000 stop not whining